client/pages/summary/[code].jsx

```diff
@@ -31,7 +31,9 @@ export default function Summary(props) {
 	var parsedData = JSON.parse(atob(props.cont));
 
 	//last item in array
-	const person = parsedData.at(-1);
+	// const person = parsedData.at(-1);
+	// The above code is what was causing issues in Safari on iOS and OSX.
+	const person = parsedData;
```

Comment:
if the requirement is the last item in the array, isn't this wrong? or am i missing some context
we should do
`const person = parsedData[parsedData.length-1]`

Reply:
Yea, that's better, just tested still works. will do changes now.